src/openforms/forms/tests/variables/test_variables_in_logic_trigger.py

```diff
@@ -117,6 +116,57 @@ def test_invalid_user_defined_variable_in_trigger(self):
             error["invalidParams"][0]["reason"],
         )
 
+    def test_invalid_date_format_for_user_defined_variable(self):
+        user = SuperUserFactory.create()
+        form = FormFactory.create()
+        FormVariableFactory.create(
+            source=FormVariableSources.user_defined,
+            key="date_var",
+            form=form,
+            data_type=FormVariableDataTypes.date,
+        )
+        FormStepFactory.create(
+            form=form,
+            form_definition__configuration={
+                "components": [
```

Comment:
Maybe you don't need to set the configuration. Since in this case it doesn't really matter what the form is?